docs/KustomizeBestPractices.md

```diff
@@ -39,38 +39,6 @@ installs/
 Defining separate packages for each component makes it easier to use composition to define new configurations; e.g. using an external database as opposed
 to a database running in cluster.
 
-## Reuse patches
```

Comment:
Can we keep this until we find a tested replacement?

Reply:
I thought it would be best not to encourage people to do exactly what we have now spent time on removing from the manifests. We wouldn't want people creating manifests that essentially cause a regression before people have had the time to play with and test [kustomize components]( https://github.com/kubernetes-sigs/kustomize/blob/master/examples/components.md).

Reply:
I agree that we want to move to components. The problem we've had issues moving to kustomize v4: https://github.com/kubeflow/manifests/issues/1797
Does it make sense to leave a note here? That we want to move to `components` as an alternative to the `load_restrictor` flag, but can't do it yet because of https://github.com/kubeflow/manifests/issues/1797.

Reply:
I believe https://github.com/kubeflow/manifests/issues/1797 can be closed as the Katib manifests have been merged. But I just pushed a commit that adds this section again with a note.

Reply:
@DavidSpek we can't close #1797 yet. Please see:
https://github.com/kubeflow/manifests/issues/1797#issuecomment-817017412

Reply:
Right, my mistake. I'll look into that a bit more as well. How does the note look?